repos/system_upgrade/el7toel8/actors/repositoriesexcluded/actor.py

```diff
@@ -0,0 +1,23 @@
+from leapp.actors import Actor
+from leapp.libraries.actor.repositoriesexcluded import process
+from leapp.models import (
+    RepositoriesExcluded,
+    RepositoriesFacts,
+    RepositoriesMap,
+)
+from leapp.reporting import Report
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class RepositoriesExcludedList(Actor):
+    """
+    Generate list of repository IDs that should be ignored by Leapp during upgrade process
+    """
+
+    name = 'repositories_excluded_list'
```

Comment:
```suggestion
    name = 'exclude_repositories'
```
The original name (repositories_blacklist) was not good at all. Because most of the actors are named using a verb (e.g. check) or noun explaining some kind of action (e.g. scanner), I think we can name it even better now. I'd go with the verb here.